src/root/components/ThreeWForm/index.tsx

```diff
@@ -348,447 +361,459 @@ function ThreeWForm(props: Props) {
         <>
-          <InputSection
-            title={strings.projectFormReportingNational}
-            description={strings.projectFormReportingHelpText}
-            tooltip={strings.projectFormReportingTooltip}
-          >
-            <SelectInput
-              error={error?.reporting_ns}
-              name={"reporting_ns" as const}
-              onChange={onValueChange}
-              options={nationalSocietyOptions}
-              pending={fetchingCountries}
-              value={value.reporting_ns}
-            />
-          </InputSection>
-          <InputSection
-              title={strings.projectFormReportingNationalContact}
-              description={strings.projectFormReportingNationalContactText}
-          >
-            <TextInput
-                name="reporting_ns_contact_name"
-                label="Name"
-                onChange={onValueChange}
-                value={value.reporting_ns_contact_name}
-            />
-            <TextInput
-                name="reporting_ns_contact_role"
-                label="Role"
-                onChange={onValueChange}
-                value={value.reporting_ns_contact_role}
-            />
-            <TextInput
-                name="reporting_ns_contact_email"
-                label="Email"
-                onChange={onValueChange}
-                value={value.reporting_ns_contact_email}
-            />
-          </InputSection>
-          <InputSection
-            title={strings.projectFormCountryTitle}
-            description={strings.projectFormCountryHelpText}
-            tooltip={strings.projectFormCountryTooltip}
-          >
-            <SelectInput
-              error={error?.project_country}
-              label={strings.projectFormCountryLabel}
-              name="project_country"
-              onChange={handleProjectCountryChange}
-              options={countryOptions}
-              pending={fetchingCountries}
-              value={value.project_country}
-            />
-            <SelectInput<"project_districts", number>
-              disabled={shouldDisableDistrictInput}
-              pending={fetchingDistricts}
-              error={error?.project_districts}
-              isMulti
-              label={strings.projectFormDistrictLabel}
-              name="project_districts"
-              onChange={onValueChange}
-              options={districtOptions}
-              placeholder={districtPlaceholder}
-              value={value.project_districts}
-              actions={(
-                <button
-                  title={strings.projectFormSelectAllDistricts}
-                  type="button"
-                  className={_cs(
-                    styles.selectAllDistrictsButton,
-                    'button button--secondary',
-                    shouldDisableDistrictInput && 'disabled',
-                  )}
+          {languageMismatch && projectDetailsResponse && (
+            <Container contentClassName={styles.languageMismatch}>
```

Comment:
Please break this container and styling in different line :
<Container 
contentClassName={styles.languageMismatch}
>